packages/session/src/store.ts

```diff
@@ -0,0 +1,10 @@
+export interface SessionStoreData {
+  data: any;
+  expiresAt: Date;
+}
+
+export interface SessionStore {
+  set(id: string, session: any, expiresAt: Date): Promise<void>;
+  get(id: string): Promise<SessionStoreData | null>;
+  destroy(id: string): Promise<void>;
+}
```

Comment:
A difference between our new implementation and `express-session`: the expiration time is a first-class thing that stores and sessions are aware of.